build/ci/evergreen.yml

```diff
@@ -24,6 +24,7 @@ variables:
       GOROOT: ${go_root}
       GOPATH: ${go_base_path}${workdir}
       ADD_PATH: "${go_bin}:${go_base_path}${workdir}/bin:${go_base_path}${workdir}/src/github.com/mongodb/mongodb-atlas-cli/bin"
+      GOPROXY: https://:@artifactory.corp.mongodb.com/artifactory/api/go/go,direct
```

Comment:
@wtrocki can you please confirm this matches the recommendation you got from proxy.golang.org team, to use the internal artifactory and fallback to `direct`?

Reply:
can you please make this an expansion + project setting?

Reply:
Looks very good.

Reply:
thanks guys